pkg/reconciler/configuration/configuration.go

```diff
@@ -127,7 +128,8 @@ func (c *Reconciler) reconcile(ctx context.Context, config *v1alpha1.Configurati
 	}
 
 	// First, fetch the revision that should exist for the current generation.
-	lcr, err := c.latestCreatedRevision(config)
+	sortedRevisions, err := c.getSortedCreatedRevisions(config)
+	var lcr *v1alpha1.Revision
 	if errors.IsNotFound(err) {
```

Comment:
The way things are written (if I'm reading correctly), this won't be true after the first Revision is created, so I don't think createRevision will be called after the first Revision is created.

I think we now also need some logic that `sortedRevisions[0].Labels[serving.ConfigurationGenerationLabelKey]` matches `metadata.generation`.

Reply:
I see, but if we add the check for generation, it wouldn't support [this use case](https://github.com/knative/serving/blob/master/pkg/reconciler/configuration/configuration_test.go#L134), right?